src/routes/safe/store/models/types/transaction.ts

```diff
@@ -32,7 +32,7 @@ export type TransactionProps = {
   baseGas: number
   blockNumber?: number | null
   cancelled?: boolean
-  confirmations: import('immutable').List<any>
+  confirmations: import('immutable').List<import('./confirmation').Confirmation>
```

Comment:
why do we use import('') here?

Reply:
Ah, this was an attempt to be able to use `.d.ts` files with imports (https://stackoverflow.com/a/51114250/2856529), but didn't work either.

So I can move them to the top, and let the file as `.ts`

Reply:
But it's not `.d.ts`, so you should be able to use regular imports

Am I missing something?

Reply:
no, I have to refactor. But didn't get the time to work on this issue yesterday.